### PR TITLE
Sdk update

### PR DESCRIPTION
**Pull request by naylin27**
https://github.com/RakutenReady/Team/issues/51742

Added CarMake, CarModel, and LicensePlate to UI
Added CSUserInfo to send name, email, number, CarMake, CarModel, and LicensePlate

### CODE REVIEW

**Review by amitmats — APPROVED** [review]

File: Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-feature android:name="android.hardware.location.gps" />
+    <uses-permission android:name="android.permission.ACTIVITY_RECOGNITION" />
```
Comment (vmewadarakuten) [inline Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml:9]: Could you help me identify where this permission is required ?

File: Android/Java/UserSessionApp/app/src/main/java/com/curbside/usersessionapp/MainActivity.java [hunk 2]
```diff
@@ -140,6 +154,22 @@ public void onClick(final View view) {
         switch (view.getId()) {
             case R.id.bRegisterTrackingId:
                 trackingIdentifier = etTrackingIdentifier.getText().toString();
+
+                //set CSUserInfo
+                name = "John Doe";
+                email = "abc@email.com";
+                number = "555-555-5555";
```
Comment (vmewadarakuten) [inline Android/Java/UserSessionApp/app/src/main/java/com/curbside/usersessionapp/MainActivity.java:161]: May want to remove hardcoded values. and thank you for updating this code.

**Review by vmewadarakuten — APPROVED** [review]

File: Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (naylin27) [reply Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml]: @vmewadarakuten In the developer docs it said to add this block of code but this line was not needed so I removed it.

File: Android/Java/UserSessionApp/app/src/main/java/com/curbside/usersessionapp/MainActivity.java (hunk 2 above)
Comment (naylin27) [reply Android/Java/UserSessionApp/app/src/main/java/com/curbside/usersessionapp/MainActivity.java]: Added fields in the UI in the app and removed the hardcoded value.

File: Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (amitmats) [reply Android/Java/UserSessionApp/app/src/main/AndroidManifest.xml]: @naylin27 Can you update the developer docs? Thanks

**Pull request merged**